dotcom-rendering/src/web/components/LabsShadyPie.tsx

```diff
@@ -0,0 +1,105 @@
+import { css } from '@emotion/react';
+import { labs, neutral, text, textSans } from '@guardian/source-foundations';
+import { Link } from '@guardian/source-react-components';
+import LabsLogo from '../../static/logos/the-guardian-labs.svg';
+
+const shadyStyle = css`
+	color: ${neutral[7]};
+	${textSans.medium({ fontWeight: 'regular' })};
+	text-decoration: none;
+	:hover {
+		text-decoration: underline;
+	}
+`;
+
+const shadyBackground = css`
+	background-color: ${neutral[97]};
+`;
+
+const shadyBanner = css`
+	background-color: ${labs[400]};
+	${textSans.xsmall({ fontWeight: 'bold' })};
+	padding: 0.5rem;
+`;
+
+const shadyTitle = css`
+	color: ${neutral[7]};
+	padding-left: 0.5rem;
+	padding-bottom: 0.5rem;
+`;
+
+const shadySponsorText = css`
+	color: ${text.supporting};
+	${textSans.xxsmall({ fontWeight: 'bold' })};
+	text-align: center;
+	padding-top: 1rem;
+`;
+
+const shadySponsorImg = css`
+	margin: auto;
+	display: block;
+`;
+
+const parentContainer = css`
+	display: flex;
+	align-items: flex-end;
+`;
+
+const labsLogoContainer = css`
+	width: 180px;
+	display: inline-block;
+	padding: 0.5rem;
```

Comment:
We use pixels in dcr. It's annoying, how do you know when to use one or the other?! It's a whole thing. I think we might align better in the future cross team / codebases but for now DCR uses pixels everywhere.